packages/l/llvm/package.yml

```diff
@@ -185,25 +191,22 @@ setup      : |
     # Support solus, it's triplet, etc.
     %patch -p1 -i $pkgfiles/0001-Support-solus-as-a-distro.patch
 
-    # https://github.com/llvm/llvm-project/issues/58219
-    %patch -p1 -i $pkgfiles/remove-dep-on-devicertl-gpu-plugins.patch
+    # AMDGPU related
+    %patch -p1 -i $pkgfiles/fix-amdgpu-si-fold-operands-assertion-error.patch
 
-    # Patch cfe for Solus specific options
     pushd clang
+        # Patch cfe for Solus specific options
         %patch -p1 -i $pkgfiles/0001-cfe-Use-correct-Solus-multilib-paths.patch
-        %patch -p1 -i $pkgfiles/0002-cfe-Use-the-GNU-hash-style-for-Solus-binary-build-id.patch
```

Comment:
I assume this is the default now?

Reply:
Yes, at least for now:
```cpp
  // Do not use 'gnu' hash style for Mips targets because .gnu.hash
  // and the MIPS ABI require .dynsym to be sorted in different ways.
  // .gnu.hash needs symbols to be grouped by hash code whereas the MIPS
  // ABI requires a mapping between the GOT and the symbol table.
  // Android loader does not support .gnu.hash until API 23.
  // Hexagon linker/loader does not support .gnu.hash
  if (!IsMips && !IsHexagon) {
    if (Distro.IsOpenSUSE() || Distro == Distro::UbuntuLucid ||
        Distro == Distro::UbuntuJaunty || Distro == Distro::UbuntuKarmic ||
        (IsAndroid && Triple.isAndroidVersionLT(23)))
      ExtraOpts.push_back("--hash-style=both");
    else
      ExtraOpts.push_back("--hash-style=gnu");
  }
```
Seems like Solus is not famous enough to get hard-coded into the clang codebase :sweat_smile: